client/src/index.js

```diff
@@ -31,6 +32,11 @@ const Routes = () => {
               Status
             </Menu.Item>
           </Link>
+          <Link className="nav-link" to="/UserRegisterForm">
```

Comment:
Can use `/sign-up-user` to match the route format and case of the interpreter route.

Reply:
ok